packages/components/pie-card-container/src/card-container.scss

```diff
@@ -1,90 +1,99 @@
 
-@mixin card-interactive-states($bg-color) {
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+@mixin card-interactive-states($bg-color, $mode: 'default') {
     &:hover:not([disabled]) {
-        --hover-modifier: calc(-1 * var(--dt-color-hover-01));
-        --card-bg-color: hsl(var(#{$bg-color}-h), var(#{$bg-color}-s), calc(var(#{$bg-color}-l) + var(--hover-modifier)));
+        @if $mode == 'inverse' {
+            --hover-modifier: var(--dt-color-hover-02);
+        } @else {
+            --hover-modifier: var(--dt-color-hover-01);
+        }
+
+        --card-bg-color: hsl(var(#{$bg-color}-h), var(#{$bg-color}-s), var(#{$bg-color}-l), var(--hover-modifier));
     }
 
     &:active:not([disabled]) {
-        --active-modifier: calc(-1 * var(--dt-color-active-01));
-        --card-bg-color: hsl(var(#{$bg-color}-h), var(#{$bg-color}-s), calc(var(#{$bg-color}-l) + var(--active-modifier)));
+        @if $mode == 'inverse' {
+            --active-modifier: var(--dt-color-active-02);
+        } @else {
+            --active-modifier: var(--dt-color-active-01);
+        }
+
+        --card-bg-color: hsl(var(#{$bg-color}-h), var(#{$bg-color}-s), var(#{$bg-color}-l), var(--active-modifier));
     }
 }
 
-// By default, the card is styled to the 'Default' variant designs
 .c-card-container {
     --card-bg-color: var(--dt-color-container-default);
     --card-radius: var(--dt-radius-rounded-c);
-    --card-shadow: var(--dt-elevation-01);
+    --card-border-color: transparent;
 
-    display: block;
+    position: relative;
     color: currentcolor;
     background-color: var(--card-bg-color);
+    border: 1px solid var(--card-border-color);
     border-radius: var(--card-radius);
-    text-decoration: none;
-    position: relative;
 
-    @include card-interactive-states('--dt-color-container-default');
+    &[variant='default'] {
+        box-shadow: var(--dt-elevation-01);
+
+        @include card-interactive-states('--dt-color-black');
+    }
+
+    &[variant='outline'] {
+        border-color: var(--dt-color-border-strong);
+
+        @include card-interactive-states('--dt-color-black');
+    }
+
+    &[variant='inverse'] {
+        --card-bg-color: var(--dt-color-container-dark);
+
+        box-shadow: var(--dt-elevation-inverse-01);
+
+        @include card-interactive-states('--dt-color-container-default', 'inverse');
+    }
+
+    &[variant='outline-inverse'] {
+        --card-bg-color: var(--dt-color-container-dark);
+
+        border-color: var(--dt-color-border-inverse);
+
+        @include card-interactive-states('--dt-color-container-default', 'inverse');
+    }
 
     &[disabled] {
         --card-bg-color: var(--dt-color-disabled-01);
+
         pointer-events: none;
-    }
 
-    // We should not apply the default elevation styling if the card is focused
-    // but we want to make sure that it is still applied when the card is clicked
-    &:not(:focus-within),
-    &:active:focus-within {
```

Comment:
I spoke to Naz about this and she said it's fine to apply the shadow for the focus state as well, thus, removing it.